src/main/java/org/folio/innreach/domain/entity/base/Auditable.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.innreach.domain.entity.base;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Getter
+@Setter
+@ToString
+@MappedSuperclass
+@EntityListeners(AuditingEntityListener.class)
+public class Auditable {
+
+  @CreatedBy
+  @Column(name = "created_by")
+  private String createdBy;
+
+  @CreatedDate
+  @Column(name = "created_date")
+  private Date createdDate;
```

Comment:
I'd suggest to use java.sql.Timestamp or even java.time.OffsetDateTime (to have timezone info) instead of java.util.Date

Reply:
I was thinking about this, thanks for the adwise, will use java.time.OffsetDateTime